src/ex_actor/internal/network.cc

```diff
@@ -134,8 +152,10 @@ void MessageBroker::SendProcessLoop(const std::stop_token& stop_token) {
       zmq::multipart_t multi;
       multi.addmem(serialized_identifier.data(), serialized_identifier.size());
       multi.add(std::move(reply_operation.data));
+      last_heartbeat_ = std::chrono::high_resolution_clock::now();
```

Comment:
use `steady_clock`, high_resolution_clock may change when system clock is modified.